aggregator/apps/api/v1/api_client.py

```diff
@@ -22,31 +25,50 @@ def get_event_loop():
     if not params:
         params = {}
 
-    params["utm_medium"] = "devs.DC API"
-    headers = {"User-Agent": "devs.DC API"}
+    headers = {"Accept": "application/json", "User-Agent": "devs.DC API"}
     timeout = aiohttp.ClientTimeout(total=10)
     async with aiohttp.ClientSession(timeout=timeout) as session:
         async with session.get(url, params=params, headers=headers) as response:
             body = await response.json()
             return {"url": url, "json": body, "status": response.status}
 
 
-class ApiClient:
-    def __init__(self):
+def proxy_single_request(loop, url, params=None):
```

Comment:
If we want to do that (and maybe throw something like a `400 Bad Request` instead of just the standard `404 Not Found`?) if you try and pass an invalid ID, I'd prefer to do that in the EE API and forward the error response.

I reckon as far as possible, we should try and keep "domain knowledge" (e.g: what is/isn't a valid election id) in the upstream APIs and keep this layer fairly "dumb" in that respect.